components/skus/renderer/skus_utils.cc

```diff
@@ -5,24 +5,37 @@
 
 #include "brave/components/skus/renderer/skus_utils.h"
 
+#include <algorithm>
+#include <string>
 #include <vector>
 
 #include "base/no_destructor.h"
 #include "third_party/blink/public/platform/web_security_origin.h"
 #include "third_party/blink/public/platform/web_url.h"
 #include "url/gurl.h"
 
-namespace skus {
+namespace {
+// NOTE: please open a security review when appending to this list.
+std::vector<GURL> safe_origins_gurl{GURL("https://account.brave.com"),
+                                    GURL("https://account.bravesoftware.com"),
+                                    GURL("https://account.brave.software")};
 
-bool IsSafeOrigin(const blink::WebSecurityOrigin& origin) {
-  // NOTE: please open a security review when appending to this list.
-  static base::NoDestructor<std::vector<blink::WebSecurityOrigin>> safe_origins{
-      {{blink::WebSecurityOrigin::Create(GURL("https://account.brave.com"))},
-       {blink::WebSecurityOrigin::Create(
-           GURL("https://account.bravesoftware.com"))},
-       {blink::WebSecurityOrigin::Create(
-           GURL("https://account.brave.software"))}}};
+base::NoDestructor<std::vector<blink::WebSecurityOrigin>>
+WebSecurityOriginList() {
```

Comment:
Can this function return just the vector, and then we leave to the caller to give the vector to a `NoDestructor` instance?

Reply:
Fixed!